README.md

```diff
@@ -14,32 +15,32 @@ This git repository is laid out as a mono-repo, containing multiple independent
 
 ![Graph of interdependencies of Pace modules, generated from dependences.dot](./dependencies.svg)
 
-### Dynamical core tests
+### Installing
 
-To run dynamical core tests, first get the test data from inside `fv3core` or `fv3gfs-physics` folder, then build `fv3gfs-integration` docker image at the top level.
+The easiest way to install the model is to create a virtual environment for it and install each component into that. This could be done with:
 
 ```shell
-$ cd fv3core
-$ make get_test_data
-$ cd ../
-$ make build
+$ python -m venv pace-venv        # Create the virtual environment
+$ source pace-venv/bin/activate   # Activate it in the shell
+$ cat install.sh                  # Always check scripts before executing them!
+$ ./install.sh                    # Script installs all components into the current env
 ```
 
-To enter the container:
-```shell
-$ make dev
-```
+### Dynamical core tests
 
-Then in the container, dynamical core serial tests can be run:
+To run dynamical core tests, first get the test data from inside `fv3core` or `fv3gfs-physics` directory, then run the tests
```

Comment:
Maybe say by default this will pull `c12_6ranks_standard`, which is the only one currently publicly available? Also, have we made the open buket?